src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -209,20 +210,21 @@ bool MulticopterLandDetector::_get_ground_contact_state()
 	// low thrust: 30% of throttle range between min and hover, relaxed to 60% if hover thrust estimate available
 	const float thr_pct_hover = _hover_thrust_estimate_valid ? 0.6f : 0.3f;
 	const float sys_low_throttle = _params.minThrottle + (_params.hoverThrottle - _params.minThrottle) * thr_pct_hover;
-	bool ground_contact = (_actuator_controls_throttle <= sys_low_throttle);
+	_has_low_throttle = (_actuator_controls_throttle <= sys_low_throttle);
```

Comment:
Is `_has_low_throttle` used for anything other than setting `ground_contact`?

Reply:
It gets logged otherwise not.

Reply:
I guess it would be enough to log only one then no?

Reply:
The two are not the same. The `ground_contact` variable is only initialized with `has_low_throttle` and later is combined with other conditions like `in_descend`. It's a bit confusing but I didn't add it. Further refactoring is definitely necessary.